docker/config.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# setup config & key files
+DIR="../appconfig/files/projectaanvraag/docker/silex/"
```

Comment:
Does this not require a change to the appconfig repo? I did not find this change.

Reply:
Yes, see this PR: https://github.com/cultuurnet/appconfig/pull/430 (I was still adding the widgetbeheer files to avoid multiple PR's with 1 file change).